src/main/java/mops/businesslogic/FileService.java

```diff
@@ -32,4 +32,17 @@
      * @return a list of file in that folder
      */
     List<FileInfo> getFilesOfDirectory(Account account, long dirId);
+
+    /**
+     * @param account user credentials
+     * @param fileId file id of needed file
+     * @return file (not int, type not known yet)
+     */
+    int getFile(Account account, long fileId);
+
+    /**
+     * @param account user credentials
+     * @param fileId file id of file to be deleted
+     */
+    void deleteFile(Account account, long fileId);
```

Comment:
this return type is also something to talk about:
return a boolean for success or use exceptions to also transmit an error message?

Reply:
In my opinion it is cleaning to leave it void and throw exception upon error.